public/js/selfoss-shares.js

```diff
@@ -33,6 +33,10 @@ selfoss.shares = {
         this.register('mail', 'e', function(url, title) {
             document.location.href = 'mailto:?body=' + encodeURIComponent(url) + '&subject=' + encodeURIComponent(title);
         });
+        this.register('copy', 'c', function(url) {
+            selfoss.copyString(url);
+            selfoss.ui.showMessage(selfoss.ui._('url_copied'));
```

Comment:
We currently do not have `selfoss.ui._`. #1137 introduces `selfoss.ui.translate`, though.

Reply:
I retract this comment. Forgot we had this.